tests/test_parse.py

```diff
@@ -27,6 +31,30 @@ def test_float(self):
         c = parse(Config, ["--a", "12", "--c", "1.23"])
         assert c.c == 1.23
 
-    def test_bool(self):
-        c = parse(Config, ["--a", "12", "--d", "True"])
+    def test_bool_0(self):
+        c = parse(Config, ["--a", "12", "--f", "0"])
+        assert c.f is False
+
+    def test_bool_1(self):
+        c = parse(Config, ["--a", "12", "--f", "1"])
+        assert c.f is True
+
+    def test_bool_lower(self):
+        c = parse(Config, ["--a", "12", "--f", "true"])
+        assert c.f is True
+
+    def test_bool_upper_false(self):
+        c = parse(Config, ["--a", "12", "--f", "False"])
+        assert c.f is False
+
+    def test_bool_upper_true(self):
```

Comment:
maybe add 1 missing test case where you enter some gibberish and expect the error